.nvmrc

```diff
@@ -0,0 +1 @@
+14.17.3
```

Comment:
More of a question, we have `14.17.3` in 3 places. I assume there isn't a way to have it in 1?

Reply:
yeah it's a good point :)
The source of truth could be package.json engines, and we extract to an env var that we can then use in docker config and scripts.
Not sure about the .nvmrc though. Maybe I'll try removing it actually and see if engines warning is enough.

Reply:
I removed the .nvmrc